ui/backend/uiserver/server.go

```diff
@@ -68,3 +71,10 @@ func (srv *uiServer) Start() {
 func (srv *uiServer) Stop() {
 	log.Print("UI Postee server stopped!")
 }
+
+func (ctx *uiServer) pingHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+	jsonStr, _ := json.Marshal("Postee UI alive!")
```

Comment:
no need to marshal json. Empty body is fine here

Reply:
done

Reply:
Done